start-lightspeed.sh

```
@@ -0,0 +1,154 @@
+    fi
+
+    # ── Create Python venv if needed ───────────────────────────────
+    if [ ! -d "$LIGHTSPEED_SERVICE_DIR/.venv" ]; then
+        echo "Creating Python virtual environment for lightspeed-service..."
+        python3 -m venv "$LIGHTSPEED_SERVICE_DIR/.venv"
+    fi
+
+    # ── Start lightspeed-service in a subshell ─────────────────────
+    echo "Starting lightspeed-service on port $OLS_PORT..."
+    (
+        cd "$LIGHTSPEED_SERVICE_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)